Tests/Test_Lights.c

```diff
@@ -17,11 +18,11 @@ int main() {
                 Lights_Read(LEFT_BLINK),
                 Lights_Read(BPS_PWR),
                 Lights_Read(BrakeLight));
-        fflush(stdout);
-        
+        fflush(stdout);  
         light_t light = rand() % 10;
         State state = Lights_Read(light);
         Lights_Set(light, !state);
+        usleep(10000);
```

Comment:
Just a note: this test does not compile for the STM32F4 target, since we can't rely on POSIX compliance.

Reply:
The tests will be target-specific. Once we have to test on the STM32 it'll be a slightly different test